apps/website/src/content/docs/keyboardkeys.mdx

```diff
@@ -1,18 +1,27 @@
 ---
 title: Keyboard keys
```

Comment:
The grammatically correct name would be "Keyboard key" not "Keyboard keys".

But the component is called `Kbd` so I'm thinking we should rename the file and title to use "Kbd" as well.